src/coreclr/jit/gentree.cpp

```diff
@@ -8652,6 +8652,15 @@ GenTree* Compiler::gtClone(GenTree* tree, bool complexOK)
             break;
         }
 
+        case GT_FTN_ADDR:
+        {
+            copy = new (this, tree->OperGet()) GenTreeFptrVal(tree->TypeGet(), tree->AsFptrVal()->gtFptrMethod);
```

Comment:
Looks like it also needs to unconditionally copy `gtFptrDelegateTarget`:
```
copy->AsFptrVal()->gtFptrDelegateTarget = tree->AsFptrVal()->gtFptrDelegateTarget;
```

Reply:
`gtCloneExpr` doesn't copy that, should that be added there too?

Reply:
Yes.

Reply:
Done